frontend/awx/resources/templates/TemplatesList.tsx

```diff
@@ -27,13 +27,18 @@ export function TemplatesList(props: {
   url?: string;
   projectId?: string | undefined;
   inventoryId?: string | undefined;
+  credentialsId?: string | undefined;
 }) {
   const { t } = useTranslation();
   const pageNavigate = usePageNavigate();
   const getPageUrl = useGetPageUrl();
   const toolbarFilters = useTemplateFilters();
   const tableColumns = useTemplateColumns();
-  const getQueryParams = (projectId?: string | undefined, inventoryId?: string | undefined) => {
+  const getQueryParams = (
+    projectId?: string | undefined,
```

Comment:
`undefined` type shouldn't be necessary if prop is optional.

Reply:
Fixed! Also removed an unnecessary undefined above this in the TemplatesList props as well.